tools-local/tasks/core-setup.tasks.csproj

```diff
@@ -34,6 +34,9 @@
   </ItemGroup>
   <ItemGroup Condition="'$(TargetGroup)' != 'net45'">
     <Compile Include="netstandard/FileUtilities.netstandard.cs" />
+
+    <Reference Include="$(BuildToolsTaskCoreDir)Microsoft.DotNet.Build.CloudTestTasks.dll" />
```

Comment:
The problem with BuildToolsTaskDir is that it is dependent on what platform you're running on, not what framework you're targeting.  If we're building on Windows, we want to be able to reference the desktop assembly and the core assembly independently.